plugins/vst_base/VstPlugin.cpp

```diff
@@ -305,45 +306,51 @@ void VstPlugin::toggleUI()
 
 void VstPlugin::setTempo( bpm_t _bpm )
 {
-	lock();
-	sendMessage( message( IdVstSetTempo ).addInt( _bpm ) );
-	unlock();
+	if (try_lock())
```

Comment:
Can we invert the control flow here and at the other places where `try_lock` is used to this:
```c++
if (! try_lock())
    return;
}
```
I think it's more natural this way because it doesn't have the normal case inside an `if` condition and as a bonus we avoid all the indentation changes, which makes reviewing easier.

Reply:
Consider it done :)